src/draw.c

```diff
@@ -380,6 +380,39 @@ static void free_layouts(GSList *layouts)
         g_slist_free_full(layouts, free_colored_layout);
 }
 
+static cairo_surface_t *render_backdrop(cairo_surface_t *srf, colored_layout *cl, colored_layout *cl_next, int y, int width, int height, bool first, bool last)
```

Comment:
Well, this is a quite long line. I guess, breaking it up should improve readability.

Reply:
What about naming the function to `render_background`? Personally, I heard "backdrop" the first time today, that I really had to use a translator and I guess it's quite uncommon.